apps/backoffice-v2/src/pages/Entity/components/DirectorsCallToAction/helpers.ts

```diff
@@ -0,0 +1,32 @@
+import { AnyObject } from '@ballerine/ui';
+
+export const getRevisionReasonsForDocument = (
+  { type, category }: AnyObject,
+  workflow: AnyObject,
+) => {
+  if (category === 'proof_of_identity' && type === 'passport')
```

Comment:
We might want to use an enum here or use values from `@ballerine/common`.

Reply:
Unfortunately i dont see an opportuninty to make it work in context of directors document. I dont think that i can specificy exact Typebox types depending on multiple conditions. Current solution is bad in current context but i dont see any quick way to make it work properly.